test pre-commit with `write` permission (#24)
r-devel/glossary: README.Rmd

pak::pak("r-devel/glossary")
```

## With `Write` permissions:

Steps to add pre-commit:

> 

Reviewer: ```suggestion
```

> 1. Do the following in the Terminal application of your machine. Do not do this in RStudio Terminal:

Reviewer: ```suggestion
This repository uses precommit. To install and run the same, follow the steps below:

```

>     1. For windows `pip3 install pre-commit --user`

Reviewer: ```suggestion
- In an external terminal run `pip3 install pre-commit --user` (for Windows) or `brew install pre-commit` (for macOS)
```

    2. For mac `brew install pre-commit` 
2. Clone the repo https://github.com/r-devel/glossary to a local folder.  
3. Go to RStudio Console and type 
```{r}
install.packages("devtools")
install.packages("precommit")

> precommit::use_precommit()

Reviewer: ```suggestion
```
